cdap-distributions/src/emr/install.sh

```diff
@@ -22,9 +22,9 @@
 # The git branch to clone
 CDAP_BRANCH=${CDAP_BRANCH:-release/4.1}
 # Optional tag to checkout - All released versions of this script should set this
-CDAP_TAG='v4.1.0'
+CDAP_TAG='v4.1.1'
```

Comment:
Actually, `CDAP_TAG=''` since there's no `v4.1.1` tag in Git, yet. This instructs the script which Git tag to checkout for configuration. Setting this to nothing means "use the HEAD of $CDAP_BRANCH" and is what should be used for development. Otherwise, changes to the configs won't be tested, as the script will always check out this tag. In the case of a non-existent tag, it'll fail the script.